cmd2/argparse_custom.py

```diff
@@ -1333,7 +1333,7 @@ def __init__(
         self.set_ap_completer_type(ap_completer_type)  # type: ignore[attr-defined]
 
     # noinspection PyProtectedMember
-    def add_subparsers(self, **kwargs: Any) -> argparse._SubParsersAction:
+    def add_subparsers(self, **kwargs: Any) -> Any:
```

Comment:
`argparse._SubParsersAction` is the correct return type.

Reply:
Fixed